apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/UrlSigner.Options.cs

```diff
@@ -47,7 +47,19 @@ public sealed class Options
             /// </summary>
             public SigningVersion SigningVersion { get; }
 
-            private Options(TimeSpan? duration, DateTimeOffset? expiration, SigningVersion version)
+            /// <summary>
+            /// The style of signed URL to generate.
+            /// Defaults to <see cref="UrlStyle.Path"/>.
+            /// </summary>
+            public UrlStyle UrlStyle { get; }
+
+            /// <summary>
+            /// The Scheme to use for the request. Usually http or https.
+            /// Defaults to https.
+            /// </summary>
+            public string Scheme { get; }
```

Comment:
This feels like it might be better in the template. As discussed, the dividing line is pretty fuzzy.
I'm happy to leave it here if you're reasonably happy that it belongs here - it's just worth you considering it again, in case. (You're closer to the code here, and probably have a better sense of this than I do.)

Reply:
Fine by me.